### DISCUSSION TITLE
Add `light` and `dark` as valid values for `set-local-storage-item`

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#2986] [Ideas]
Considering `My Filters` is now untrusted by default and setting site themes is one of the original use case presented for introducing `set-local-storage-item` in https://github.com/uBlockOrigin/uBlock-issues/discussions/2697, I think the values `light` and `dark` would be nice additions to `set-local-storage-item` for users using permanent incognito mode/private browsing.

Several use cases for considerations:
1. Setting light theme for FotMob (`fotmob.com`) currently requires using  
   `fotmob.com##+js(trusted-set-local-storage-item, mode, '"light"')`
3. Setting dark theme for Kaggle notebooks (`kaggle.com`) currently requires using  
   `kaggle.com##+js(trusted-set-local-storage-item, KAGGLE_THEME, dark)`
5. Setting dark theme for ChatGPT (`chat.openai.com`) currently requires using  
   `chat.openai.com##+js(trusted-set-local-storage-item, theme, dark)`

**Comment by gorhill** [comment]
We will have to put a stop on requests to support more values, otherwise what's the limit? Might as well import `words.txt`? Given the specified use case, just use `trusted-set-local-storage-item`, what you are proposing is not going to be used in a filter list, it's a case of personal preference.
